src/components/SearchBar.tsx

```diff
@@ -233,7 +234,12 @@ export function SearchBar({
     executeEntityPreviewsQuery
   ] = useEntityPreviews(entityPreviewSearcher, entityPreviewsDebouncingTime);
   const { verticalKeyToResults, isLoading: entityPreviewsLoading } = entityPreviewsState;
-  const entityPreviews = renderEntityPreviews?.(entityPreviewsLoading, verticalKeyToResults, handleSubmit);
+  const entityPreviews = renderEntityPreviews?.(
+    entityPreviewsLoading,
+    verticalKeyToResults,
+    handleSubmit,
```

Comment:
Would you be ok with moving `handleSubmit` and `getAriaLabel` into an object? then somebody could just spread them into their DropdownItem, and we could also add more props here in the future instead of continually growing the number of arguments

Reply:
In that case, would I have to change the names of it so it can be properly spread? (`handleSubmit` -> `onClick`, `getAriaLabel` -> `ariaLabel`)

Reply:
yes!